cmd/src/lsif_upload.go

```diff
@@ -71,6 +72,7 @@ Examples:
 	flags.ignoreUploadFailures = flagSet.Bool("ignore-upload-failure", false, `Exit with status code zero on upload failure.`)
 	flags.uploadRoute = flagSet.String("upload-route", "/.api/lsif/upload", "The path of the upload route.")
```

Comment:
Should we add 'internal use only' here too?

Reply:
Sure, if someone's changed this in a forked Sourcegraph instance they'll happily ignore that instruction.